docs/source/usage/common_scenarios.rst

```diff
@@ -69,4 +69,52 @@ In order for the lessee to access a node on a private network, they can do one o
     openstack esi trunk add network --tagged-networks <private network> <trunk name>
     openstack esi node network attach --trunk <trunk name> <node>
 
+Using External Networks
+-----------------------
+
+The recommended way to allow external network access to a baremetal node is by `creating an external provider network`_. The instructions in the link explain how; in addition, this network should be created as ``--provider-network-type vlan`` with the appropriate vlan specified by ``--provider-segment``.
+
+Once that's done, a lessee can provision a node upon their provisioning network and then gain external network access using one of two methods.
+
+Floating IP
+~~~~~~~~~~~
+
+In order to use a floating IP, start by creating a router for the external network and configuring it for your provisioning network:
+
+  .. prompt:: bash $
+
+    openstack router create external-router
+    openstack router set --external-gateway <external network> external-router
+    openstack router add subnet external-router <provisioning subnet>
+
+Once that's done, you can create a floating IP and associate it with the node's port (which can be found by running ``openstack esi node network list``):
+
+  .. prompt:: bash $
+
+    openstack floating ip create <external network>
+    openstack floating ip set --port <port> <floating ip>
```

Comment:
Does this step happen after the node is provisioned with provisioning network?

Reply:
Yep!